### PR TITLE
enum members can have values

### PR DESCRIPTION
**Pull request by wehrstedt**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface DeclarationBase {
 export interface EnumMemberDeclaration extends DeclarationBase {
     kind: "enum-value";
     name: string;
+    value?: any;
```
Comment (RyanCavanaugh) [inline lib/index.ts]: Should be of type `string | number`

**Comment by wehrstedt** [comment]
@RyanCavanaugh wasn't shure because in javascript you can also declare objects and any other stuff as an enum member. But you are right, thats not possible in typescript

**Comment by RyanCavanaugh** [comment]
Thanks!

**Comment by wehrstedt** [comment]
can you create a new version so that a I can use the changes?

**Comment by RyanCavanaugh** [comment]
Published 0.1.21

**Pull request merged**